app/lib/email/send-magic-link-email.js

```diff
@@ -57,5 +66,6 @@ async function sendVetMagicLinkEmail (request, email, data) {
 module.exports = {
   sendFarmerApplyLoginMagicLink,
   sendFarmerClaimLoginMagicLink,
-  sendVetMagicLinkEmail
+  sendVetMagicLinkEmail,
+  getToken
```

Comment:
I don't think adding `getToken` to the interface of `send-magic-link-email` makes sense. It would be better to have it as a separate module and then it can be imported into this one.

Reply:
ok. Make sense.